questions/26-v-model/index.test.ts

```diff
@@ -0,0 +1,19 @@
+import { mount } from "@vue/test-utils"
+import { describe, it, expect } from "vitest"
+
+import App from "./App.vue"
+
+describe("v-model", () => {
+  it("should work", async() => {
+    const wrapper = mount(App)
+    const input = wrapper.find("input")
+    const p = wrapper.find("p")
```

Comment:
Why we need this, the element seems doesn't existed

Reply:
maybe we can add it to template:
```vue
<template>
  <input v-oh-model="value" type="text" />
  <p>{{ value }}</p>
</template>
```
I forgot to commit my changes here.😅

Reply:
Thanks for your contribution .